src/symmetric/message_hash/sha.rs

```diff
@@ -47,24 +47,26 @@ where
         randomness: &Self::Randomness,
         message: &[u8; MESSAGE_LENGTH],
     ) -> Vec<u8> {
-        let mut hasher = Sha3_256::new();
-
-        // first add randomness
-        hasher.update(randomness);
-
-        // now add the parameter
-        hasher.update(parameter);
-
-        // now add tweak (= domain separator + epoch)
-        // domain separator: this is a message hash tweak.
-        hasher.update([TWEAK_SEPARATOR_FOR_MESSAGE_HASH]);
-        hasher.update(epoch.to_le_bytes());
-
-        // now add the actual message to be hashed
-        hasher.update(message);
-
-        // finalize the hash, and take as many bytes as we need
-        let hash = hasher.finalize();
+        // Setup hasher
+        let hasher = Keccak256Hash;
+
+        // Collect all input bytes
```

Comment:
```suggestion
        // Collect all input bytes in the following order
```

just to make it more explicit